### DISCUSSION TITLE
Service Worker not find (and offline not working)

### DISCUSSION THREAD

**Discussion by comrade-warm** [vuepress/core#1406] [Q&A] [answered]
When using the PWA2  plugin or the official PWA - the server worker is not found, which is why the application does not work offline on iOS (unlike viewpress v1 - it works there).

I used iOS 16.6 for testing and 
1) https://www.pwabuilder.com/reportcard?site=https://plugin-pwa2-demo.vuejs.press/
2) https://www.pwabuilder.com/reportcard?site=https://www.vuepress.cn/

How to solve this problem?

**Comment by Mister-Hope** [comment, marked as answer]
There is a typo in the demo and I fixed it in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/dd335fb7e6899fad55c15a9d41cdf871c09dbdea just now.


At least https://plugin-pwa2.vuejs.press/ should work as expected, and I can even install it via chrome throgh google play.
